From 748dca871d1f8a06d33a1002e1dfabb74b16c9b6 Mon Sep 17 00:00:00 2001
From: Chris Wright <chrisw@sous-sol.org>
Date: Thu, 1 Mar 2012 15:25:14 -0800
Subject: [PATCH] ovsdbmonitor: move to its own data directory

It doesn't have dependencies on any packages that would install into
pkgdatadir, and for Fedora it makes directory ownership work better.
So move ovsdbmonitor python internals to its own data directory.

Signed-off-by: Chris Wright <chrisw@sous-sol.org>
Signed-off-by: Ben Pfaff <blp@nicira.com>
---
 debian/ovsdbmonitor.install    |    2 +-
 ovsdb/ovsdbmonitor/automake.mk |    2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/debian/ovsdbmonitor.install b/debian/ovsdbmonitor.install
index 4f19ecb..0572be4 100644
--- a/debian/ovsdbmonitor.install
+++ b/debian/ovsdbmonitor.install
@@ -1,2 +1,2 @@
 usr/bin/ovsdbmonitor
-usr/share/openvswitch/ovsdbmonitor
+usr/share/ovsdbmonitor
diff --git a/ovsdb/ovsdbmonitor/automake.mk b/ovsdb/ovsdbmonitor/automake.mk
index 59d2e2c..ec63cc7 100644
--- a/ovsdb/ovsdbmonitor/automake.mk
+++ b/ovsdb/ovsdbmonitor/automake.mk
@@ -28,7 +28,7 @@ EXTRA_DIST += \
 	ovsdb/ovsdbmonitor/ovsdbmonitor.in
 MAN_ROOTS += ovsdb/ovsdbmonitor/ovsdbmonitor.1
 
-ovsdbmonitordir = ${pkgdatadir}/ovsdbmonitor
+ovsdbmonitordir = ${datadir}/ovsdbmonitor
 if BUILD_OVSDBMONITOR
 noinst_SCRIPTS += ovsdb/ovsdbmonitor/ovsdbmonitor
 ovsdbmonitor_DATA = $(ovsdbmonitor_pyfiles)
-- 
1.7.7.4
